contrib/nsis-installer-windows/galaxy-windows-installer

```diff
@@ -103,6 +103,15 @@ printf "\n%s\n" "WRAPPING makensis localy in $(pwd)"
 	sed -i '/START_FILES/r tmp.out' "${SCRIPT_DIR}/installer.nsi"
 	rm tmp.out
 
+	# download openvpn installer
+	URL="https://swupdate.openvpn.org/community/releases/openvpn-install-2.4.2-I601.exe"
+	wget "$URL" -P "${SCRIPT_DIR}" --no-check-certificate || { echo "Can not download $URL" ; exit 1 ; }
```

Comment:
We have our fail function. Instead of:  
echo "mesage"; exit 1; 
use:
fail "message"